### ISSUE TITLE
port wdio-visual-regression-service to v5

### ISSUE THREAD

**Issue by abjerstedt** [webdriverio/webdriverio#3082]
Describe the solution you'd like
Currently, the v5 branch does not support the wdio-visual-regression-service package. I would like to see this package ported to v5 in order to maintain compatibility.

**Comment by abjerstedt** [comment]
I'm about to start on this

**Comment by christian-bromann** [comment]
I don't mind porting it to this repo. Doing so however requires maintenance. I am not familiar with this project so an other option is to just make a PR to the original project. With that being said, feel free to port it here but if so you need to make the commitment to be responsible for all issues around that package :-P

**Comment by emilyrohrbough** [comment]
@abjerstedt There are a few issues / enhancements I would like to see addressed if you are picking up this work.

**Comment by emilyrohrbough** [comment]
Also, the wdio-visual-regression-service project is dependent on the [wdio-screenshot](https://github.com/zinserjan/wdio-screenshot) project. Are you going to work on compatibility for both?

**Comment by abjerstedt** [comment]
So, after I made this issue I talked to my boss and he is a little reluctant (didn't say no  though) for me to take ownership of this tool due to the time sync and the fact that my non of my devs are using it yet.

However, it does seem like a tool that would be better off owned by an organization such as the wdio community than  a single person that only seems to release every 6-12 months (and its currently been 8 months).

@emilyrohrbough you interested in tag teaming this? I can throw up a branch for us to work through together. From our previous conversations, until this is ported, your company won't be able to upgrade to v5.

**Comment by seanpoulter** [comment]
Have we (in the community) reached out to @zinserjan to discuss how they'd like to support the two repos? 🤔 As a dev looking to recommend an open-source visual regression testing tool it'd be lovely to see the services "officially" supported.

**Comment by abjerstedt** [comment]
@seanpoulter https://github.com/zinserjan/wdio-visual-regression-service/issues/85

This issue was logged in August and hasn't seen any response.

**Comment by emilyrohrbough** [comment]
@seanpoulter The maintainer of those repos hasn't been involved since April-ish and his last communication was minimal on a PR. 

@abjerstedt  Sorry for the delayed response. I have spoken with my team and we unfortunately are not at capacity to pick up this work for webdriver.io v5.

**Comment by seanpoulter** [comment]
Yep, Jan-André (@zinserjan) has no GitHub activity since May. Have you tried anything off GitHub @abjerstedt to make sure they're OK? You're both members of the [WebdriverIO](https://github.com/orgs/webdriverio/people) organization. 😉

**Comment by abjerstedt** [comment]
@seanpoulter I've never interacted with the him. Christian just made me a member like 8 days ago! 

Anyways, as I mentioned earlier - I haven't fully decided if i want to port this over personally. I'm allowed to spend working hours contributing to WDIO, but it has to be to the benefit of my company; and as of yet, non of my developers have taken up usage of the tool.

If you are interested, i can help you get setup to port this... but be warned - you will spend SIGNIFICANT time getting this package up to UT code coverage standards.

**Comment by seanpoulter** [comment]
8 day? **Congratulations!** 🎉

You're not using it the service, and I haven't even got it working yet. 😆I've just found it today exploring open source options for visual regression testing. 

When's v5 set to ship? I'd be game to pick this up in Feb if we haven't found folks to move this forward.

**Comment by christian-bromann** [comment]
> When's v5 set to ship?

By end of this year
